.github/workflows/ci.yml

```diff
@@ -11,52 +11,105 @@ on:
     branches:
       - "**"
 jobs:
-  check-dart:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        sdk: [stable, beta, dev]
+  check-lint-flutter:
+    name: Lint (flutter)
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
     steps:
-      - name: Checkout the code
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup flutter
+        uses: subosito/flutter-action@v2
+        with:
+          cache: true
+      - name: Lint
+        run: flutter format --output=none --set-exit-if-changed packages/flutter
+  check-lint-dart:
+    name: Lint (dart)
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
         uses: actions/checkout@v3
       - name: Setup dart
         uses: dart-lang/setup-dart@v1
+      - name: Lint
+        run: dart format --output=none --set-exit-if-changed packages/dart
+  check-code-analysis-flutter:
+    name: Code analysis (flutter)
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup flutter
+        uses: subosito/flutter-action@v2
         with:
-          sdk: ${{ matrix.sdk }}
+          cache: true
+      - name: Install dependencies
+        run: |
+          cd packages/dart
+          flutter pub get
```

Comment:
@fischerscode shouldn't this be `dart pub get`?

Reply:
It should work.
I think dart would work, too. But I'm not sure if subosito/flutter-action also installs the dart command.

Reply:
OK so I leave it as it is?

Reply:
I would leave it like this.